functions.php

```diff
@@ -935,9 +935,13 @@ function addToMailQueue($mysqli, $data) {
         $recipient = strval($email['recipient']);
         $recipient_name = strval($email['recipient_name']);
         $subject = strval($email['subject']);
-        $body = strval($email['body']);
+        $body = mysqli_real_escape_string($mysqli,strval($email['body']));
 
-        mysqli_query($mysqli, "INSERT INTO email_queue SET email_recipient = '$recipient', email_recipient_name = '$recipient_name', email_from = '$from', email_from_name = '$from_name', email_subject = '$subject', email_content = '$body'");
+        mysqli_query(
+            $mysqli,
+            "INSERT INTO email_queue (email_recipient, email_recipient_name, email_from, email_from_name, email_subject, email_content) 
+            VALUES ('$recipient', '$recipient_name', '$from', '$from_name', '$subject', '$body')"
+        );
```

Comment:
Put this back the way it was its easier to read by using set var = '$var_name' and its standardized everywhere in the code.